src/tests/ftest/util/slurm_utils.py

```diff
@@ -169,8 +174,8 @@ def check_slurm_job(handle):
                 one extra UNKNOWN if the handle doesn't match a known
                 slurm job.
     """
-    cmd = "scontrol show job {}".format(handle)
-    output = subprocess.check_output(cmd, stderr=subprocess.STDOUT, shell=True)
+    cmd = ["scontrol", "show", "job", handle]
```

Comment:
Do we need a full path for the `scontrol` command?

Reply:
not needed.

Reply:
not needed.